common/static/common/templates/discussion/new-post-alert.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="alert alert-error pattern-library-shim" role="alert" aria-labelledby="alert-title-error" tabindex="-1">
+    <span class="icon alert-icon fa fa-warning pattern-library-shim" aria-hidden="true"></span>
+
+    <div class="alert-message pattern-library-shim">
+        <h3 class="sr-only alert-title pattern-library-shim" id="alert-title-error">
```

Comment:
This id won't be unique if there are two discussion components shown in the same unit. I don't know that this is very common, but can it use a unique id?